Representable/representable
# drive page + signup user flow updates

a couple changes made to make user flow more smooth
**changes**
- updated the drive page to contain more information about representable + redistricting info for the state the drive is within
- changed user flows to redirect to signup page rather than login initially
- adapted links to ensure that "?next=" part of url is maintained between signup and login pages
- cleaned up frontend of signup and login pages
- removed michigan specific views/urls code

**to test**
- go to drive url
- check no errors / typos
- log out
- return to drive page or click on state from landing page
- check that you are redirected to signup page
- check that switching between signup and login pages maintains "?next=" part of url

**screenshots**
drive page
![image](https://user-images.githubusercontent.com/41943646/95092036-2c01c600-06f5-11eb-88c1-e3a04710ba57.png)
